extensions/helix/package.build.ts

```diff
@@ -274,6 +274,225 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
+            args: [
+              {
+                code: [
+                  `
+                  const editor = vscode.window.activeTextEditor;
+                  const fallback = () => vscode.commands.executeCommand(
+                      'workbench.action.quickOpen',
+                    );
+
+                  if (!editor) {
+                    return await fallback();
+                  }
+
+                  const currentFileUri = editor.document.uri;
+                  const currentDirectoryUri = vscode.Uri.joinPath(currentFileUri, '..');
+
+                  const workspaceFolder = vscode.workspace.getWorkspaceFolder(currentFileUri);
+                  if (!workspaceFolder) {
+                    return await fallback()
+                  }
+
+                  const relativeDirectoryPath = vscode.workspace.asRelativePath(
+                      currentDirectoryUri,
+                      false
+                    );
+
+                  const quickOpenPrefix = relativeDirectoryPath.endsWith('/')
+                    ? relativeDirectoryPath
+                    : \`\${relativeDirectoryPath}/\`;
+
+                  await vscode.commands.executeCommand(
+                    'workbench.action.quickOpen',
+                    quickOpenPrefix
+                  );
+
+                  `.split("\n").map(line => line.trim()).join(""),
+                ],
+              },
+            ],
+          },
+          "b": {
+            text: "Open buffer picker",
+            command: "workbench.action.showAllEditors",
+          },
+          "s": {
+            text: "Open symbol picker",
+            command: "workbench.action.gotoSymbol",
+          },
+          "S": {
+            text: "Open workspace symbol picker",
+            command: "workbench.action.showAllSymbols",
+            args: [],
+          },
+          "d": {
+            text: "Open diagnostic picker",
+            command: "workbench.action.problems.focus",
+          },
+          "D": {
+            text: "Open diagnostic picker",
+            command: "workbench.action.problems.focus",
+          },
+          "g": {
+            text: "Open changed file picker",
+            command: "workbench.view.scm",
+          },
+          "a": {
+            text: "Perform code action",
+            command: "editor.action.quickFix",
+          },
+          "G": {
+            text: "Debug",
+            command: "workbench.action.debug.start",
+          },
+          "w": {
+            text: "Window",
+            command: "dance.openMenu",
+            args: [{ menu: "window" }],
+          },
+          "y": {
+            text: "Yank selections to clipboard",
+            command: "dance.selections.saveText",
+            args: [{
+              register: "",
+            }],
+          },
+          "Y": {
+            text: "Yank main selection to clipboard",
+            command: "dance.selections.saveText",
+            args: [{
+              register: "",
```

Comment:
It might be best to be explicit?
```suggestion
              register: "dquote",
```